lib/accent/schemas/version.ex

```diff
@@ -22,4 +23,13 @@ defmodule Accent.Version do
     |> validate_required(@required_fields)
     |> unique_constraint(:tag, name: :versions_tag_project_id_index)
   end
+
+  def with_parsed_tag(version) do
+    trimmed_tag = String.trim_leading(version.tag, "v")
```

Comment:
Maybe yours is more readable but we could have piped it.

```suggestion
version.tag
|> String.trim_leading("v")
|> Version.parse()
|> case do
  {:ok, tag} -> %{version | parsed_tag: tag}
   _ -> %{version | parsed_tag: :error}
end
```